java-core/google-cloud-core/src/main/java/com/google/cloud/ServiceOptions.java

```diff
@@ -199,6 +201,17 @@ public B setHost(String host) {
       return self();
     }
 
+    /**
+     * Universe Domain is the domain for Google Cloud Services. It follows the format of
```

Comment:
It's unclear What "It" is that has the format in next line.

Reply:
It reads:

> A is B. It has the format "xxx.{A}".

A's format uses A in it.

Reply:
Sounds good, will update.